specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/examples/DiskMigrations/Create.json

```diff
@@ -0,0 +1,85 @@
+{
+  "parameters": {
+    "subscriptionId": "04666444-56f0-4d4f-afc5-dbd6b134b084",
+    "location": "redmond",
+    "migrationId": "ec9f79cf-e4d7-4d37-926a-e69c2198be8a",
+    "api-version": "2018-07-30-preview",
+    "targetShare": "\\\\SU1FileServer.mydomain.com\\SU1_ObjStore_1",
+    "disks": [
+      {
+        "id": "/subscriptions/04666444-56f0-4d4f-afc5-dbd6b134b084/providers/Microsoft.Compute.Admin/locations/redmond/disks/423e4eb4-f791-4d13-ad5b-4d415733b0d6",
+        "name": "redmond/423e4eb4-f791-4d13-ad5b-4d415733b0d6",
+        "type": "Microsoft.Compute.Admin/locations/disks",
+        "location": "redmond",
```

Comment:
Are these tracked resources or proxy? If proxy, no need to have "location"

Reply:
I think proxy should have location. No proxy resource doesn't need the location.